velox/functions/prestosql/BinaryFunctions.h

```diff
@@ -422,6 +422,20 @@ struct ToIEEE754Bits64 {
   }
 };
 
+template <typename T>
+struct FromIEEE754Bits64 {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<double>& result,
+      const arg_type<Varbinary>& input) {
+    static constexpr auto kTypeLength = sizeof(int64_t);
+    VELOX_USER_CHECK_EQ(input.size(), kTypeLength, "Expected 8-byte input");
+    memcpy(&result, (const uint64_t*)input.data(), kTypeLength);
```

Comment:
nit: Use `reinterpret_cast<uint64_t>(input.data())`

Reply:
`const` has no use here.

Reply:
Without `const` the compiler does not let me cast.